src/index.ts

```diff
@@ -1,10 +1,12 @@
 import 'reflect-metadata'
 import {createKoaServer} from "routing-controllers"
-import Controller from "./controller"
+import UserController from './users/controller'
+// import Controller from "./controller"
```

Comment:
So we need to get rid of the controller in the src/

Reply:
yesss